internal/cache/file/cache_handle_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/storageutil"
 	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/operations"
 	. "github.com/jacobsa/ogletest"
+	"github.com/stretchr/testify/suite"
```

Comment:
Aren't we removing full dependency from `github.com/jacobsa/ogletest` ?

Reply:
removed github.com/jacobsa/ogletest import